packages/core/src/publicCredential/PublicCredential.chain.ts

```diff
@@ -180,10 +210,13 @@ export async function credentialFromChain(
       'Block should always contain the full credential, eventually.'
     )
   }
+
+  const [credentialInput, submitter] = lastRightCredentialCreationCall
```

Comment:
Would it be nice to have one mapping less in the object below?
```suggestion
  const [credentialInput, attester] = lastRightCredentialCreationCall
```